app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/item/SpecialSessionItem.kt

```diff
@@ -15,6 +19,25 @@ data class SpecialSessionItem(
     override fun bind(viewBinding: ItemSpecialSessionBinding, position: Int) {
         viewBinding.session = session
         viewBinding.isShowDayNumber = isShowDayNumber
+
+        val now = Date()
+        when {
+            now.before(session.startTime) -> {
```

Comment:
Thanks! 
Can you add session.isOnGoing and use it?
And can you do it in xml with DataBinding?

Reply:
Yeah, I challenge it!